src/code42cli/cmds/users.py

```diff
@@ -226,6 +228,24 @@ def change_organization(state, username, org_id):
     _change_organization(state.sdk, username, org_id)
 
 
+@users.command()
+@click.argument("username")
+@click.argument("alias")
+@sdk_options()
+def add_alias(state, username, alias):
+    """Add a cloud alias for a given user."""
```

Comment:
I'm curious if theres a use case to add more than one alias at a time, there's functionality to provide multiple click options at once (https://click.palletsprojects.com/en/8.0.x/options/#multiple-options) but if the CLI is mainly aimed towards doing this in bulk, then this should be fine.

Reply:
Users can have a maximum of two aliases, and have one by default that is the same as their username. I think that we should probably keep this method using a single alias both in order to keep parity with the console UI, and also to make it less likely that users will try to add more aliases than are allowed.

Reply:
Got it, I didn't know we had a two maximum, then yes I agree this is probably the right methood.